### DISCUSSION TITLE
2008/08/14/%E6%8C%96%E8%8B%A6%E4%B8%AD%E5%9B%BD%E7%94%B7%E8%B6%B3%E7%9A%84

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#237] [Announcements]
# 2008/08/14/%E6%8C%96%E8%8B%A6%E4%B8%AD%E5%9B%BD%E7%94%B7%E8%B6%B3%E7%9A%84

今天LabVIEW讨论群里还没讨论技术，大家就先发了一堆讽刺男足的贴。我整理了一下：

https://blog.qizhen.xyz/2008/08/14/%E6%8C%96%E8%8B%A6%E4%B8%AD%E5%9B%BD%E7%94%B7%E8%B6%B3%E7%9A%84

**Comment by ruanqizhen** [comment]
之前博客下的留言：

秋
2008年08月20日 09:53 编辑
網丄佷誃亽嘟侞茈說：
 
紾噯甡掵，逺離啯哫~
 
吥褦吥叹蒠錒~~~

回复
Xiaohong
2008年08月20日 09:43 编辑
看了真是大快人心！日前看到一网上奥运竞猜题目，问若中国男足参加女足比赛，结果是什么？大部分人的回答是仍然不能小组出线。

回复
tscwl
2008年08月17日 12:20 编辑
唉，写得忒好！！

回复
